skills/odl-pdf/scripts/quick-eval.py

```
@@ -0,0 +1,281 @@
+        snippets.append("\n".join(current_hunk))
+
+    return snippets
+
+
+def build_report(
+    extracted_path: Path,
+    ground_truth_path: Path,
+    score: float,
+    method: str,
+    threshold: float,
+    verbose: bool,
```

Ruff (0.15.9)

[warning] Boolean-typed positional argument in function definition

(FBT001)